### PR TITLE
Release v2.0.0 to Production

### PR DESCRIPTION
**Pull request by bogdanadrianmarc**
This PR releases app version `2.0.0` to production

### CODE REVIEW

**Review by jonrandahl — COMMENTED** [review]
LGTM! 👍 

Will approve closer to release date

File: .nvmrc
Comment (joescottdave) [inline .nvmrc:1]: I wonder if the practice here shouldn't be to specify only the major version (numerically) e.g.
```
20
```
This is the equivalent of specifying `lts/hydrogen` but makes specifying base images in docker simpler (for example)

@jonrandahl @bogdanadrianmarc @DanielCouzens

Comment (jonrandahl) [reply .nvmrc:1]: According to the [node releases table](https://nodejs.org/en/about/previous-releases#looking-for-latest-release-of-a-version-branch) the name assigned to the v20 version is `lts/iron`, or `v20.18.0`, not the version specified here; @bogdanadrianmarc is there an alpine version for `lts/iron`?

Comment (jonrandahl) [reply .nvmrc:1]: Nevertheless, if we change things to resolve this we should do that on another branch after this is released up the stack! 😄

Comment (bogdanadrianmarc) [reply .nvmrc:1]: @joescottdave I like this idea, it's the best choice between specifying the full version vs specifying the LTS name and having to do extra steps in Dockerfile. I'll make the change for UKHPI and will need to change the other repos that use node as well.

@jonrandahl There is but will need to do extra steps in the Dockerfile as the name of the image is different. I think Joseph's idea is really good as there is an image `20-alpine3.20` which has the latest 20 node version, and judging by the date it was added this always gets replaced with the latest 20 version (currently on 20.18).

This also allows us to use any 20 version locally without having to always update to the latest.

To note, we cannot do this for ruby / rails as their release process is a bit different and they don't have the concept of LTS

Comment (jonrandahl) [reply .nvmrc:1]: > This also allows us to use any 20 version locally without having to always update to the latest.

This makes the most sense to me! Thanks for the clarification @bogdanadrianmarc! 👍

Comment (joescottdave) [reply .nvmrc:1]: > This also allows us to use any 20 version locally without having to always update to the latest.

Yes, had this in mind also, and is not actually a change from our current practice of using the `lts/<name>`

**Review by jonrandahl — APPROVED** [review]
LGTM! 👍
Can you please update the NEW spreadsheet and project board to reflect these updates? 🙏

**Pull request merged**